en/Frequency-based.md

```diff
@@ -0,0 +1,14 @@
+## Practice Recommendations Organized by Use Case
+
+### Frequency-based trips
+
+A frequency-based trip does not follow a fixed schedule but attempts to maintain predetermined headways. These trips are denoted in [GTFS frequency.txt](https://gtfs.org/reference/static/#frequenciestxt) by setting `exact_times=0` or omitting the `exact_times` field. There are several best practices to keep in mind when constructing GTFS-realtime feeds for frequency-based trips.
+
+In [TripUpdate.StopTimeUpdate](#StopTimeUpdate), the [StopTimeEvent](#StopTimeEvent) for `arrival` and `departure` should not contain `delay` because frequency-based trips do not follow a fixed schedule. Instead, `time` should be provided to indicate arrival/departure predictions.
+
+Also in [TripUpdate.StopTimeUpdate](#StopTimeUpdate), `schedule_relationship` should not be `SCHEDULED` or `SKIPPED` because these values are only relevant for schedule-based trips (i.e., trips not defined in [frequencies.txt](https://gtfs.org/reference/static/#frequenciestxt) or trips defined in [frequencies.txt](https://gtfs.org/reference/static/#frequenciestxt) but with `exact_times=1`). As of May 2019, there is no enumeration value in`schedule_relationship` that is suitable for frequency-based trips. A new enumeration value, `UNSCHEDULED`, is being proposed to the official specification and will be the best option for frequency-based trips once approved. Until `UNSCHEDULED` is approved, we recommend leaving `schedule_relationship` empty.
+
+When describing `trip` in [TripUpdate](#TripUpdate) or [VehiclePosition](#VehiclePosition) by using [TripDescriptor](#TripDescriptor), all of `trip_id`, `start_time`, and `start_date` must be provided, and`schedule_relationship` should be `UNSCHEDULED`.

```

Comment:
Usage of "must" indicates this belongs in the spec.

Reply:
I double-checked the spec and it does actually require this now. I think it's work keeping frequency-based trip recommendations/requirements in one spot, though, as current they are scattered throughout the spec on individual fields. So I labeled this "As required by the spec, ..." in https://github.com/MobilityData/GTFS_Realtime_Best-Practices/pull/1/commits/022a3e6e0ba99379fe3e7be2741c9fd6f55addc3. We can discuss further and determine if this type of info belongs in Best Practices.

Reply:
Syncromatics supported having this in one spot too (see https://mobilitydata-io.slack.com/archives/C3D321CKB/p1619644777021900?thread_ts=1619643803.018800&cid=C3D321CKB):

>I think it’s definitely helpful to have them collected together in the best practices. It’s a very specific use case with a set of practices that are otherwise distributed. As long as it can generally be kept up to date with changes, I think it’s worth repeating for the sake of organizing them together.

Reply:
Note, however, that there is a similar section in the "Trip-Updates.md" markdown file in the google/transit repo (but not within reference.md):
https://github.com/google/transit/blob/master/gtfs-realtime/spec/en/trip-updates.md#systems-with-repeated-trip_ids

We should ultimately merge these two pieces of content into one that's easy to understand.